doc/cla/corporate/syncordia.md

```diff
@@ -0,0 +1,16 @@
+Ireland, 2015-06-26
+
+Syncordia Technologies and Healthcare Solutions Ireland Limited agrees to the terms of the 
+Odoo Corporate Contributor License Agreement v1.0.
+
+I declare that I am authorized and able to make this agreement and sign this
+declaration.
+
+Signed,
+
+Vadim Zenin vadim.zenin@syncordiahealth.ie https://github.com/Vadim-Zenin
+
+List of contributors:
+
+Vadim Zenin vadim.zenin@syncordiahealth.ie https://github.com/Vadim-Zenin
+Vadims Briksins vadim.bryksin@syncordiahealth.ie https://github.com/Briksins
```

Comment:
1) "Image" didn't work out, it must be 'image_medium'
2) Master to where I am suppose to commit the change is Odoo V9, but I was doing fix specifically for Odoo V8 so it seems I did right to commit to branch 8.0. anyway doing everything the way you asked